source/meta.tex

```diff
@@ -2498,7 +2498,7 @@
 \tcode{N} and \tcode{D}:
 
 \begin{itemize}
-\item \tcode{num} shall have the value \tcode{sign(N) * sign(D) * abs(N) / gcd}.
+\item \tcode{num} shall have the value \tcode{$\operatorname{sgn}(\tcode{N})$ * $\operatorname{sgn}(\tcode{D})$ * abs(N) / gcd}.
```

Comment:
Do we need _this_ many nested `\tcode`s? Can't it just be a single, all-encompassing `\tcode`?

Reply:
I think so, due to the math font. Grepping the sources for `'tcode.*\$'` reveals many such cases. Perhaps `\operatorname` can be used without math font.

Reply:
Ah, but then, the parentheses of _sgn_ would be in code font, but _sgn_ is a math function, not a code entity.

Reply:
Ah yes, makes sense, thanks!